README.md

```diff
@@ -15,29 +22,19 @@ It deploys:
   - An access policy to allow the MI to read secrets
   - an SSH key and uploads
 
+- An `aci-devops-agents` module that creates:
+  - data structures for the vnet/subnet created before
+  - an `azurerm_network_profile` and `azurerm_container_group` that deploy a container running the Azure DevOps agents in the vnet (which requires delegation to `Microsoft.ContainerInstance/containerGroups`)
+  - The agents autamatically register themsleves with Azure DevOps.
 
+  The pipeline in `src/pipelines/infra.yaml` applies the content on the `main.tf` in `src/terrafomr` which calls the `common` module and the `aci-devops-agents` module in sequence; after that, a call to the terrafom template in `src/terraform/delete_access_policy` eliminates the access policy for the `current` service connection (SC) service principal (SP) effectively removing access to the secret to subsequent Azure DevOps runs. This way, even by accessing the SC' SP will make it impossible to access the secrets in Keyvault.
```

Comment:
typo: `src/terraform`

Reply:
I suggest breaking this down into a numbered list of actions that happen in sequence, for clarity's sake. Also suggest using full words instead of acronyms in the last sentence even if they are already explained, in the previous sentence.